### ISSUE TITLE
ergoCub 1.1 S/N:001 – Fingers cannot reach the upper limits and have jerky motions

### ISSUE THREAD

**Issue by PasMarra** [robotology/icub-tech-support#1703]
### Robot Name 🤖

ergoCub 1.1 S/N:001

### Request/Failure description

While testing the aperture and closure of the fingers, I've noticed the following:

- Jerky movements when there is a zero position reference.
- `thumb_add` maximum reachable value is `90 deg`, but for the [documentation](https://icub-tech-iit.github.io/documentation/hands/hands_mk5/) is `100 deg`.
- several fingers cannot reach maximum value.

### Detailed context

I'll provide some plots, each one shows the desired trajectory vs the joint actual trajectory for both the hands, the joints values are in radian.

### Additional context

# thumb_add

![thumb_add](https://github.com/robotology/icub-tech-support/assets/143645988/58f0e0e1-abdc-4b0f-a53a-9c40acd4c0f6)

![thumb_add_zoom](https://github.com/robotology/icub-tech-support/assets/143645988/b251549e-6877-41b9-b3c7-8bb3bcd5f888)
*thumb_add*

# thumb_oc

![thumb_oc](https://github.com/robotology/icub-tech-support/assets/143645988/b0edfb10-3cab-4daf-9570-29414f687665)
*thumb_oc*

# index_add

![index_add](https://github.com/robotology/icub-tech-support/assets/143645988/e1e24d29-8c44-4066-9e3c-6433e8061597)
*index_add*

# index_oc

![index_oc](https://github.com/robotology/icub-tech-support/assets/143645988/d1119548-c23f-465c-b0f4-226e747f501f)
*index_oc*

# middle_oc

![middle_oc](https://github.com/robotology/icub-tech-support/assets/143645988/6e6884d4-870b-43c0-b702-b2a7316722b2)
*middle_oc*

# ring_pinky_oc

![ring_pinky_oc](https://github.com/robotology/icub-tech-support/assets/143645988/23352be0-509b-4901-8c0e-7a073fea7e9d)
*ring_pinky_oc*

**Comment by AntonioConsilvio** [comment]
Hi @PasMarra, thanks for the feedback.
We are already aware of the problems present on the fingers and have an internal issue where we are planning how to take corrective action.

**Comment by maggia80** [comment]
Thanks, @PasMarra for the detailed analysis.  it's interesting to see also the motor position, the PWM output and the current while performing the movements.

**Comment by lornat75** [comment]
@maggia80 it is not a problem for us to run the experiments - but it seems the problem is already known and under investigation from iCub Tech. are you sure you need additional data?

**Comment by maggia80** [comment]
@lornat75 yes, as @AntonioConsilvio mentioned we know that there are some problems in the fingers and we are working on it. In the experiments above I see different issues and some more data could help iCub Tech and @PasMarra too to have more awareness of the system. 
But of course, those are data we can collect and report here in this issue.

**Comment by AntonioConsilvio** [comment]
Closing due this comment:

- https://github.com/robotology/icub-tech-support/issues/1819#issuecomment-2616180058
